tools/spinxsk.ps1

```diff
@@ -221,6 +221,9 @@ while (($Minutes -eq 0) -or (((Get-Date)-$StartTime).TotalMinutes -lt $Minutes))
         if ($LastExitCode -ne 0) {
             throw "SpinXsk failed with $LastExitCode"
         }
+    } catch {
```

Comment:
How was this fix tested? In the original bug #810 we can see that the error in the screenshot occurs within the `finally` block and is intentionally suppressed with `-ErrorAction 'Continue'`.

AFAIK, all errors within the `try` block cause the script to exit already.

Reply:
Actually, maybe that screenshot shows an install failure. I think the problem may be https://github.com/microsoft/xdp-for-windows/blob/181c77437b37a8e8e0b58880e95fbb7439c82a54/tools/setup.ps1#L823-L825

which defaults to Continue, it seems, even though the caller (spinxsk.ps1) has set its own `$ErrorActionPreference` to Stop. I'm not a Powershell wizard, but the original intent would have been for an error emitted within the setup.ps1 script to cause the spinxsk.ps1 script to bail when the `-ErrorAction` parameter is not supplied.

Reply:
I was planning on drafting 2 PRs off this PR (stacked PR)

1 in which I change the script to invoke an error (containing this PR's fix) and confirm the CI is crashing
1 in which I change the script to invoke the same error but without the fix, and confirm the CI is passing (when it shouldn't)

Reply:
@mtfriesen please see: 

https://github.com/microsoft/xdp-for-windows/actions/runs/14846993171/job/41683239218?pr=841
and 
https://github.com/microsoft/xdp-for-windows/actions/runs/14846939995/job/41683069550?pr=840

Reply:
Wow, baffling behavior. Thanks for verifying.